# $DragonFly: src/secure/usr.sbin/sshd/Attic/log.h.patch,v 1.1 2004/08/30 21:59:58 geekgod Exp $
--- log.h_orig	2004-08-22 16:25:25.000000000 +0000
+++ log.h	2004-08-22 16:25:38.000000000 +0000
@@ -15,6 +15,8 @@
 #ifndef SSH_LOG_H
 #define SSH_LOG_H
 
+#define __dead __dead2
+
 #include <syslog.h> /* Needed for LOG_AUTHPRIV (if present) */
 
 /* Supported syslog facilities and levels. */
